class_defines.py

```diff
@@ -2276,6 +2277,8 @@ def needs_wp(self):
 
     def use_flipping(self):
         # Only use flipping for constraint between point and line/workplane
+        if self.entity1.is_curve:
```

Comment:
is_curve is a function, has to be called

Reply:
fixed